proto/api/v1/tenant.proto

```diff
@@ -398,3 +404,20 @@ message TenantServiceUpdateMemberResponse {
   // TenantMember is the updated membership
   TenantMember tenant_member = 1;
 }
+
+// TenantServiceRequestAdmissionRequest is used to request admission for a tenant
+message TenantServiceRequestAdmissionRequest {
+  // Login of the tenant for whom admission is requested
+  string login = 1;
+  // Name of the user
+  string name = 2;
+  // Email of the user
+  string email = 3 [(buf.validate.field).string.email = true];
+  // AcceptedTermsAndConditions indicates if the user has accepted the terms and conditions
+  bool accepted_terms_and_conditions = 4;
+  // EmailConsent indicates if the user gave consent to receive emails
+  bool email_consent = 5;
```

Comment:
Mail Consent is used only in the hubspot service, which we can't use because that needs a PaymentCustomer.


``err = s.hubspotService.CreateNewHubspotContact(ctx, req.Customer, req.EmailConsent)
	if err != nil {
		s.log.Error("unable to send contact to hubspot", "error", err)
	}``